### PR TITLE
py_rmpe_server is in working condition

### PR DESCRIPTION
**Pull request by anatolix**
I've finished python version of rmpe_server.
It is 4.5 times faster than C++ version(140 pic/s vs 30 pics/sec on my server)
code is approx. 2 times shorter and it is python, not c++

I made pull request not to immediately join it to main repo, at this point I want somebody except me looks on my code, test it, try to train model, etc.

update: during training loss is becoming NaN, need to find a bug 
update2: fixed

### CODE REVIEW

**Comment by michalfaber** [comment]
Thanks @anatolix. This is really good work. Did you obtain similar results for COCO dataset as in original implementation?

**Comment by anatolix** [comment]
I've made stupid bug with batch size in the beginning, I set it to 50, and for my surprise doesn't get oom, so i kept it. But actually iterator still returned 10, this killed learning rate too early and I have to retrain again. 
I started again this morning now I have loss 505.8195 after 10 epochs and 10 hours of training, it looks like working, but I need several days to reach 400-something like pretrained network. I will write here when I will get it.

**Comment by anatolix** [comment]
Stupid question, did code in C++ augmentation achieve original code performance?
I did training for 25epochs, L2 loss(17) is better than converted network(20), L1 is almost same(51) vs 50.
But performance of model is bad, looks almost exactly like picture in your readme file (arm is missing)

Losses calculated on 2645 items validation **unaugmented** dataset. 
For canonical model c++ code was used, for mine my python code. augmentation was disabled in both programs.

**Comment by anatolix** [comment]
Trained on original C++ rmpe_server, model is working better on ski.jpg, while reporting it's loss is actually worse. Will be looking for difference in augmentations.

**Comment by michalfaber** [comment]
C++ augmentation is almost the same as in original implementation. Even number generator is taken from caffe. I didn’t compare concrete results from both implementations but I would say that it should be very similar.

**Comment by anatolix** [comment]
Comparation of rmpe_server and py_rmpe_server results(TLDR: both worse than original model)
https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/issues/39
